cpp/ps0000/problem 0.cpp

```diff
@@ -1,14 +1,15 @@
-#include <iostream>
+#include <iostream> // 2/2
 // first mistake : @ ,, # not #
 // second mistake : "iostream.h" ,, should be iostream without .h and ""
 using namespace std;
 //forgot this line
-int main()
-// he write void and we don't need it ,, Main instead main
+int main() // 1/1
+// Main instead main
 {
-  cout << "Hello World \n " ;
+  cout << "Hello, World\n"; // 3/4
 // first mistake >> ,, << not >>
 // second mistake /n ,, \n not /n
-  return 0;
+  return 0; // 1/1
 // return 0 not return 1
-}
\ No newline at end of file
+}
+// great work 😎 10/10 ⭐

```

Comment:
- [x] most of code errors and bugs are fixed
- [x] file is in the right path
- [x] +1 mark for your first commit
+1 bonus mark for your comments ⭐